backup-recovery.adoc

```diff
@@ -1,26 +1,101 @@
+
+Then delete the whole `Bisq` folder and replace it with the `Bisq` folder from your backup.
+
+=== Details
+
+In your Bisq data directory, you will see a few items. The most notable file is `bisq.log`, which is the Bisq program log file. Checking it can be helpful when you have issues with the Bisq program.
+
+The other notable item is `btc_mainnet`, which is the main folder containing data for your payment accounts, Tor setup, and wallet.
+
+== Payment Accounts
+
+There are 2 components to payment accounts on Bisq: account details and account aging.
+
+**Account details** consist of all the payment methods you've set up and their associated account numbers, bank information, addresses, etc. You can export this data from within Bisq on the `National currency accounts` and `Altcoin accounts` tabs of the `Account` screen. Click `Export accounts` and Bisq will output a file that you can use to import these details later.
+
+**Account aging** establishes the age of your payment accounts. To reduce fraud, newer payment accounts have lower trading limits.
+
+To include account aging with your account details import, replace the `btc_mainnet/keys/sig.key` file with the `sig.key` file from your backup. Be sure you don't have any open offers, trades, or disputes before changing `sig.key`, otherwise you won't be able to respond to those activities.
+
+If you don't have an accounts export file, but you have a full backup of the data directory, you can still fully restore your accounts details and accounts aging by manually copying these files to your data directory from a backup:
+
+* `btc_mainnet/db/AccountAgeWitnessStore` (salt)
+* `btc_mainnet/keys/sig.key` (key used to sign p2p messages)
+* `btc_mainnet/keys/UserPayload` (account details)
+
+**Again, please make sure you don't have any open offers, trades, or disputes before changing `sig.key`.**
+
+IMPORTANT: Before manually making any changes to Bisq data files, please remember to back up the current state of your Bisq data directory and make sure the Bisq program is closed.
+
+== Tor State
+
+Bisq runs as a Tor hidden service to protect your privacy.
+
+As you do trades, you'll build _local reputation_ whereby Bisq will show you how many trades you've completed with other peers. The idea is that if one trade with someone went well, it's reasonable to expect your next trade with that person will go well too, creating a rudimentary sort of reputation while maintaining anonymity. This mechanism is tied to your onion address.
+
+If you want to migrate a particular onion address, you can replace the `btc_mainnet/tor/hiddenservice` folder in your data directory with the one from your backup.
+
+If you want to change your onion address, just delete the `btc_mainnet/tor/hiddenservice`. The next time you start Bisq, you'll have a different onion address.
```

Comment:
Maybe add a note that you gain more privacy with that as past trades are not connected with new trades or offers via the onion address (only visible to past traders) but u will lose the reputation.